libs/model/package.json

```diff
@@ -30,8 +30,9 @@
     "@hicommonwealth/core": "workspace:*",
     "@hicommonwealth/schemas": "workspace:*",
     "@hicommonwealth/shared": "workspace:*",
-    "@solana/web3.js": "^1.91.6",
+    "@knocklabs/node": "^0.6.11",
```

Comment:
Do we need `@knocklabs/node` as a dependency here given it is not directly referenced anywhere in `model/`

Reply:
Huh.  Let me yank it and find out.